4-storage/kinesis-lzo-s3-sink/README.md

```diff
@@ -0,0 +1,87 @@
+# Kinesis LZO S3 Sink
+
+## Introduction
+
+The Kinesis LZO S3 Sink consumes records from an [Amazon Kinesis][kinesis] stream, compresses them using [splittable LZO][hadoop-lzo], and writes them to S3.
```

Comment:
Can we add a sentence about how we handle _framing_ (where one message ends and the next begins)? Explain how we use the Elephant Bird project's `BinaryBlockWriter`, and the Protocol Buffer array. Will save somebody a lot of pain...